diff -ar -U 1 --unidirectional-new-file base/.config/bspwm/bspwmrc laptop/.config/bspwm/bspwmrc
--- base/.config/bspwm/bspwmrc	2019-06-11 17:36:50.155274640 +1000
+++ laptop/.config/bspwm/bspwmrc	2019-06-11 17:39:48.311499117 +1000
@@ -23,3 +23,2 @@
 
-
-### vbox ###
+xautolock -time 5 -locker "i3lock -n -i ~/.config/background.png" -killtime 10 -killer "systemctl suspend" -detectsleep &
diff -ar -U 1 --unidirectional-new-file base/.config/sxhkd/sxhkdrc laptop/.config/sxhkd/sxhkdrc
--- base/.config/sxhkd/sxhkdrc	2019-06-09 15:14:52.669272038 +1000
+++ laptop/.config/sxhkd/sxhkdrc	2019-06-11 17:37:28.421864576 +1000
@@ -131,2 +131,15 @@
 
-### laptop keys ###
+XF86AudioRaiseVolume
+    amixer -q sset Master,0 1%+ unmute
+
+XF86AudioLowerVolume
+    amixer -q sset Master,0 1%- unmute
+
+XF86AudioMute
+    amixer -q sset Master,0 toggle
+
+XF86MonBrightnessUp
+    light -A 5
+
+XF86MonBrightnessDown
+    light -U 5
